src/SonarQube.Client/Models/SonarQubeProjectBranch.cs

```diff
@@ -27,12 +27,14 @@ public class SonarQubeProjectBranch
         public string Name { get; }
         public bool IsMain { get; }
         public DateTimeOffset LastAnalysisTimestamp { get; }
+        public string Type { get; set; }
```

Comment:
Why not enum?

Reply:
We do not have the list of all possible values. Also we do not need to maintain the values since we do not care other than short.